test/infrastructure/docker/go.mod

```diff
@@ -7,20 +7,20 @@ require (
 	github.com/go-logr/logr v0.1.0
 	github.com/onsi/ginkgo v1.10.3
 	github.com/onsi/gomega v1.7.1
-	github.com/pkg/errors v0.8.1
+	github.com/pkg/errors v0.9.0
+	gopkg.in/yaml.v3 v3.0.0-20191120175047-4206685974f2
 	k8s.io/api v0.0.0-20191121015604-11707872ac1c
-	k8s.io/apimachinery v0.0.0-20191121015412-41065c7a8c2a
+	k8s.io/apimachinery v0.17.0
 	k8s.io/client-go v11.0.0+incompatible
 	k8s.io/klog v1.0.0
 	sigs.k8s.io/cluster-api v0.2.6-0.20191223162332-fd807a3d843b
 	sigs.k8s.io/cluster-api/test/framework v0.0.0-20191216153338-db9baf9ade0c
 	sigs.k8s.io/controller-runtime v0.4.0
-	sigs.k8s.io/kind v0.6.1
+	sigs.k8s.io/kind v0.7.0
```

Comment:
Can we use v0.7 with k8s 1.16 clusters?

Reply:
yeah, this has no impact on what version of clusters are supported since we are only using the types (to pass to `kind` the binary). Both 6.1 (the one `ensure-kind.sh` uses) and 7.0 are fine versions to have. The important line is getting rid of the replace to force v5.1

Reply:
perfect!